Integrations/src/main/java/io/deephaven/integrations/learn/Future.java

```diff
@@ -34,7 +36,7 @@
         this.inputs = inputs;
         this.colSets = colSets;
         this.batchSize = batchSize;
-        this.rowSet = RowSetFactory.empty();
+        this.rowSetBuilder = RowSetFactory.builderRandom();
```

Comment:
Are you certain you need random-order support, rather than `builderSequential()`?

Reply:
Discussed, answered, needs random.